<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>520</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>sscom for linux</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>317</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>串口号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="serialPortInfoListBox">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="baudRateBox">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>数据位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dataBitsBox">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="stopBitsBox">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>校验位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="parityBox">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>流  控</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="flowControlBox">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>315</y>
      <width>21</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="receive_textBrowser">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>10</y>
      <width>610</width>
      <height>261</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>266</y>
      <width>621</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>305</y>
      <width>901</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>318</y>
      <width>471</width>
      <height>181</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>471</width>
      <height>181</height>
     </size>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>0</y>
       <width>471</width>
       <height>182</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="openserial_pushButton">
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>27</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="2">
       <widget class="QCheckBox" name="hexsend_checkBox">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>HEX发送</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QCheckBox" name="rts_checkBox">
        <property name="text">
         <string>RTS</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> qproperty-wordWrap: true;</string>
        </property>
        <property name="text">
         <string>ms/次</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="9">
       <widget class="QLineEdit" name="sendLineEdit"/>
      </item>
      <item row="0" column="5" colspan="2">
       <widget class="QCheckBox" name="hexdisplay_checkBox">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>HEX显示</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="dtr_checkBox">
        <property name="text">
         <string>DTR</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="3">
       <widget class="QLineEdit" name="timer_lineEdit">
        <property name="inputMethodHints">
         <set>Qt::ImhFormattedNumbersOnly</set>
        </property>
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="led_label">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>21</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> qproperty-wordWrap: true;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/led/:/off</pixmap>
        </property>
       </widget>
      </item>
      <item row="4" column="4" colspan="2">
       <widget class="QPushButton" name="send_pushButton">
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>27</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" colspan="2">
       <widget class="QCheckBox" name="newLineCheckBox">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>发送新行</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="5">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="minimumSize">
         <size>
          <width>212</width>
          <height>148</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="timersend_checkBox">
        <property name="text">
         <string>定时发送</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> qproperty-wordWrap: true;</string>
        </property>
        <property name="text">
         <string>字符串输入框:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>SSCOM FOR LINUX</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>282</y>
      <width>617</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="openfile_pushButton">
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>打开文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="openfile_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendfile_pushButton">
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>发送文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_pushButton">
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_pushButton">
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>保存窗口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_pushButton">
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>清除窗口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="help_pushButton">
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>帮助</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dash_pushButton">
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>—</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ext_pushButton">
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>扩展</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
